ang/crmMailing/FromAddress.js

```diff
@@ -18,7 +18,10 @@
           var addr = crmFromAddresses.getByLabel(newValue);
           mailing.from_name = addr.author;
           mailing.from_email = addr.email;
-          mailing.replyto_email = addr.email;
+          // CRM-18364: set replyTo as from_email only if custom replyTo is disabled in mail settings.
+          if (!CRM.crmMailing.enableReplyTo) {
+            mailing.replyto_email = crmFromAddresses.getByAuthorEmail(mailing.from_name, mailing.from_email, true).label
```

Comment:
Also my reading of the replyto column would suggest all we need is the email not the full label here.